baseapp/baseapp_test.go

```diff
@@ -510,15 +510,20 @@ func TestValidatorChange(t *testing.T) {
 
 	// Assert that validator updates are correct.
 	for _, val := range valSet {
+
+		pk, err := tmtypes.PB2TM.PubKey(val.PubKey)
 		// Sanity
-		assert.NotEqual(t, len(val.PubKey), 0)
+		assert.Nil(t, err)
 
 		// Find matching update and splice it out.
 		for j := 0; j < len(valUpdates); {
 			valUpdate := valUpdates[j]
 
+			upk, err := tmtypes.PB2TM.PubKey(valUpdate.PubKey)
```

Comment:
upk is a confusing variable name - `pk2` or `pkUpdated` make more sense to me

Reply:
Changed to `pubkeyUpdate`